test/decl/init/resilience.swift

```diff
@@ -16,54 +16,49 @@ extension Point {
 
 // Size is not @_fixed_layout, so we cannot define a new designated initializer
 extension Size {
+  // FIXME: Produce a decent diagnostic here
   init(ww: Int, hh: Int) {
-  // expected-error@-1 {{initializer declared in an extension of non-'@_fixed_layout' type 'Size' must delegate to another initializer}}
     self.w = ww
-    self.h = hh
+    self.h = hh // expected-error {{}}
```

Comment:
What message do we get at the moment? It's worth including just so we can see when it changes.

Reply:
We now get a Sema error saying that 'self.h' is a let property. The initializer is now delegating since it's inlinable, so we don't even get as far as DI here, instead we hit the restriction that delegating inits cannot assign to let properties at all.

This isn't the "bad diagnostic" case I was talking about; that's in the DI tests (look at the patch that makes all enum constructors delegating).

Reply:
That should still be included, though. We can fix it later.

Reply:
The enum thing does seem like something we can fix in a follow-up PR. We just need to not ship it!